lib/src/StaticFileRouter.cc

```diff
@@ -235,7 +235,14 @@ void StaticFileRouter::route(
             }
         }
     }
-    callback(HttpResponse::newNotFoundResponse());
+    if (defaultHandler_)
+    {
+        defaultHandler_(req, std::move(callback));
+    }
+    else
+    {
+        callback(HttpResponse::newNotFoundResponse());
+    }
 }
```

Comment:
There are several `newNotFoundResponse` methods called in StaticFileRouter, why only this one was replaced?

Reply:
For no reason, I didn't go through the whole file.

I changed the way it's done to simplify code, I hope it's ok.